### DISCUSSION TITLE
Milvus execption - messahe=syncTimeStamp - failed: context deadline exceeded)

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#19993] [Q&A and General discussion]
Milvus exception - message=syncTimeStamp - failed: context deadline exceeded)
When connection through pymilvus (on Hyper-V virtual machine).
I get following error - 

pymivlus.exceptions.MilvusException Milvus exception - ( code=1, message=syncTimeStamp - failed: context deadline exceeded)

**Comment by yhmo** [comment]
Need more information.
Please create an issue in the [issues](https://github.com/milvus-io/milvus/issues) panel. And describe in what situation you encountered the problem. Milvus version is also needed.
